scripts/installer/default.config

```diff
@@ -1,7 +1,7 @@
 [glassfish]
 HOST_DNS_ADDRESS = localhost
-GLASSFISH_USER = glassfish
-GLASSFISH_DIRECTORY = /usr/local/glassfish4
+GLASSFISH_USER = dataverse
+GLASSFISH_DIRECTORY = /usr/local/payara5
```

Comment:
I almost touched `doc/sphinx-guides/source/_static/util/default.config` as part of the doc issue I'm working on but stopped after seeing the edit above. There are many copies of `default.config` (screenshot below) and we should figure out where/when to update them.

<img width="532" alt="Screen Shot 2020-04-13 at 2 36 41 PM" src="https://user-images.githubusercontent.com/21006/79149033-80566980-7d94-11ea-8395-d92d9a118c94.png">

Reply:
Note that these other copies are a different config file. Must be an older style docker config? - the format is different/incompatible with what the python installer is using. 

I should probably rename the new, configparser-style file.